fe/fe-core/src/main/java/com/starrocks/catalog/PhysicalPartitionImpl.java

```diff
@@ -107,6 +111,16 @@ public long getId() {
         return this.id;
     }
 
+    @Override
+    public String getName() {
```

Comment:
will it be a null or "" if upgraded from an older version?

Reply:
it's null